app/models/event.py

```diff
@@ -187,7 +187,7 @@ def __init__(self,
         self.payment_country = payment_country
         self.payment_currency = payment_currency
         self.paypal_email = paypal_email
-        self.call_for_papers = call_for_papers
+        self.speakers_calls = speakers_calls
```

Comment:
One event can and should be related to only one speakers_call. So make this singular.